.ci/checker-framework.groovy

```diff
@@ -435,17 +444,68 @@ class CheckerFrameworkError implements Comparable<CheckerFrameworkError> {
             return i
         }
 
-        i = getMessage() <=> other.getMessage()
-        if (i != 0) {
-            return i
+        if (this.isUnstable() || other.isUnstable()) {
+            final String messageWithoutLineNumber = getMessage().replaceAll('\\d+', '')
+            final String thatMessageWithoutLineNumber = other.getMessage().replaceAll('\\d+', '')
```

Comment:
Why didn't we do this in the constructor? This is suppose to be an immutable class.

Would also save on execution time, as this gets run probably every time a new entry gets added to the set for nearly everything in the set multiple times.

We obviously don't care about the numbers anyways.

Reply:
> This is suppose to be an immutable class

It still is, the original field is not modified.

> Would also save on execution time, as this gets run probably every time a new entry gets added to the set for nearly everything in the set multiple times.

It will only run for unstable mutations, only a few of them are there.

> We obviously don't care about the numbers anyways.

I feel it would be better if we don't modify the original error itself.